pages/api/logout/index.ts

```diff
@@ -0,0 +1,34 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getSessionIdFromCookies } from '../../../src/utils/session';
+import axios from 'axios';
+
+const Logout = async (req: NextApiRequest, res: NextApiResponse) => {
+  const sessionCookie = getSessionIdFromCookies(req);
+  if (sessionCookie) await logoutAdmin(sessionCookie);
+
+  res.setHeader(
+    'Set-Cookie',
+    `session_id=deleted; Path=/; secure; HttpOnly; SameSite=Strict; expires=Thu, 01 Jan 2003 00:00:00 GMT`,
+  );
+  const logoutUrl =
+    process.env.ONE_LOGIN_ENABLED === 'true'
```

Comment:
I'm not sure there is a scenario where the user can logout if One Login is disabled.  In that case we shouldn't display the logout link

Reply:
Admins can still logout from their admin session from find. We could render it for admins only?